common/djangoapps/student/models.py

```diff
@@ -294,6 +294,10 @@ class Meta(object):
     bio = models.CharField(blank=True, null=True, max_length=3000, db_index=False)
     profile_image_uploaded_at = models.DateTimeField(null=True, blank=True)
 
+    @property
+    def get_arrow_url(self):
+        return "{}{}".format(settings.ARROW_URL, settings.COUNTRY_LIST.get(self.country.name, ''))
```

Comment:
You should use urlparse lib to make correct url
`import urlparse`
`urlparse.urljoin(settings.ARROW_URL, settings.COUNTRY_LIST.get(self.country.name, ''))`

Reply:
changed